assertion/function/assertiontree/preprocess_blocks.go

```diff
@@ -91,61 +89,6 @@ func copyGraph(graph *cfg.CFG) *cfg.CFG {
 	return newGraph
 }
 
-// stripNoops returns a copy of the passed slice `effects`, minus any no-ops
```

Comment:
Code starting here is simply moved to `rich_check_effect.go`, no logic is really changed